packages/cli-repl/src/connect-info.spec.ts

```diff
@@ -74,15 +75,17 @@ describe('getConnectInfo', function() {
 
   it('reports on an enterprise version >=3.2 of mongodb with credentials', function() {
     const output = {
-      isAtlas: true,
-      isLocalhost: false,
-      serverVersion: '3.2.0-rc2',
-      isEnterprise: true,
-      authType: 'LDAP',
-      isDataLake: false,
-      dlVersion: null,
-      isGenuine: true,
-      serverName: 'mongodb',
+      is_atlas: true,
+      is_localhost: false,
+      server_version: '3.2.0-rc2',
+      is_enterprise: true,
+      auth_type: 'LDAP',
+      is_data_lake: false,
+      dl_version: null,
+      is_genuine: true,
+      non_genuine_server_name: 'mongodb',
+      server_arch: 'x86_64',
+      server_os: 'osx',
       uri: ATLAS_URI
```

Comment:
What does `URI` look like? Something like this: `mongodb+srv://<credentials>@cluster0-e06dc.azure.mongodb.net/test?authSource=admin&replicaSet=Cluster0-shard-0&readPreference=primary&ssl=true`? Might be PII, not sure if we can collect it.

Reply:
yes, that's what it looks like. I do not send this to telemetry though, only to user's local log file